### PR TITLE
Install semantic ui

### PR DESCRIPTION
**Pull request by dkuku**
I created a demo component and it's loaded in the index.js file, instead the wink.js. It can be deleted later. I left it for testing.
As you see the changes from @Kachulio1 are merged here. I made only some small changes to the readme and index.js.

This pr needs to be added after the semantic-ui assets - then it will be only a couple of files. or check the commits one by one - it's only few items.

### CODE REVIEW

**Review by mattwr18 — APPROVED** [review]
Looking good @dkuku, this will be really cool to start using

File: README.MD [hunk 1]
```diff
@@ -1 +1,18 @@
-[![Coverage Status](https://coveralls.io/repos/github/AgileVentures/WebsiteOne-FE/badge.svg?branch=develop)](https://coveralls.io/github/AgileVentures/WebsiteOne-FE?branch=develop)
\ No newline at end of file
+[![Coverage Status](https://coveralls.io/repos/github/AgileVentures/WebsiteOne-FE/badge.svg?branch=develop)](https://coveralls.io/github/AgileVentures/WebsiteOne-FE?branch=develop)
+
+# Semantic ui - customize
```
Comment (joaopapereira) [inline README.MD]: Is there a way to automate this, so that each person so not need to do this?

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Can we please remove the packge-lock.json I found this
`The solution is to install it using npm and then run yarn install to update your yarn.lock file.` in https://jsramblings.com/2018/03/04/how-to-use-semantic-ui-with-a-custom-theme-in-your-CRA-app.html

To be fair this issue would enough for me to no to use this library

**Comment by dkuku** [comment]
I made a pr with compiled assets-no one needs to do this. I made the manual
in the  readme just for reference

File: README.MD (hunk 1 above)
Comment (joaopapereira) [reply README.MD]: Can we move this to a section that is called.
**Rebuild the semantic UI asserts**

**Comment by joaopapereira** [comment]
@dkuku can you please check why tests are failing? Thanks

**Review by joaopapereira — APPROVED** [review]
Looks good, going to merge

**Pull request merged**